components/remote_settings/src/client.rs

```diff
@@ -659,15 +676,15 @@ pub struct RemoteSettingsRecord {
     pub last_modified: u64,
     #[serde(default)]
     pub deleted: bool,
-    pub attachment: Option<Attachment>,
+    pub attachment: Option<AttachmentMetadata>,
     #[serde(flatten)]
     pub fields: RsJsonObject,
 }
 
 /// Attachment metadata that can be optionally attached to a [Record]. The [location] should
 /// included in calls to [Client::get_attachment].
 #[derive(Clone, Debug, Deserialize, Serialize, Eq, PartialEq, uniffi::Record)]
-pub struct Attachment {
+pub struct AttachmentMetadata {
```

Comment:
I like this name better, but technically it's a breaking change for our consumers.   Can you do a couple things for this?
- Call out the change in `CHANEGLOG`
- Test building firefox-ios and firefox-android with these changes and see if anything breaks

Reply:
* Added a `CHANGELOG`
* Tested it on `firefox-ios` and could built it with the local `app-services`

Reply:
Reverted the change.